qutebrowser/config/configtypes.py

```diff
@@ -50,7 +50,8 @@
 import functools
 import operator
 import json
-import typing
+from typing import (Any, Callable, Dict as DictType, Iterable, Iterator,
+                    List as ListType, Optional, Pattern, Sequence, Tuple, Union)
```

Comment:
`Dict` and `List` have a name collision with the classes in this module. I have therefore imported them as `DictType` and `ListType` and I felt it was more visually consistent rather than using `typing.List`, and it saves a few characters. Happy to change it if you feel differently.